generators/client/files-angular.js

```diff
@@ -111,6 +110,13 @@ const files = {
             templates: [
                 'blocks/interceptor/_auth.interceptor.ts'
             ]
+        },
+        {
+            condition: generator => !generator.skipServer,
```

Comment:
But if there are mores stuff that would require such check lets not bother and lets just document that if you skip server its your responsibility to provide equivalent API